src/SWP/Bundle/FixturesBundle/Resources/themes/DefaultTheme/FOSUserBundle/views/layout.html.twig

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
```

Comment:
it is for popup window; for confirmation link target we extend our theme's base.html.twig